dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jakarta-connectionManagement-1.0.feature

```diff
@@ -6,5 +6,5 @@ IBM-Provision-Capability:\
 IBM-Install-Policy: when-satisfied
 -features=com.ibm.websphere.appserver.jakarta.connector-2.0
 -bundles=com.ibm.ws.jca.cm.jakarta
-kind=noship
+kind=beta
```

Comment:
auto features cannot be beta due to some quirks with the build process. Instead, move auto features to `kind=ga` (they won't be included in the GA image until the features requiring enablement are also GA)

Reply:
That's odd, but I'll switch the pull so that it uses GA for the beta of the autofeatures as you point out.

Reply:
@aguibert that doesn't sound right, i'm sure i've done beta auto features before

Reply:
There is an issue whereby you can't have a beta auto-feature where all the features it depends on are GA ... but that doesn't seem to be the case here. Even if it was, the usual solution is to introduce a dummy/temporary beta public feature and depend on that as well.

Reply:
In this case, the autofeatures depend on a transaction-2.0 feature which is beta.  I'll give this a try (in a separate commit, just in case we need to fall back to the ga approach) and see if it makes it through a build that way.